common/src/test/scala/uk/ac/wellcome/test/ModelsTest.scala

```diff
@@ -0,0 +1,31 @@
+package uk.ac.wellcome.platform.common
+
+import org.scalatest.{FunSpec, Matchers}
+
+import uk.ac.wellcome.models.Work
+import uk.ac.wellcome.utils.JsonUtil
+
+
+class WorkTest extends FunSpec with Matchers {
```

Comment:
I decided to break them into distinct test classes, because I expect to be adding tests that are specific to `Work` soon.